### DISCUSSION TITLE
How does the regular Expression deals with conflicts?

### DISCUSSION THREAD

**Discussion by NanGuoBean** [ActivityWatch/activitywatch#731] [Q&A] [answered]
When Categorizing, if one matches two different regular expressions, which rules would it follow?
For example, I put keyword "youtube.com" in category "Videos", and keyword "python" in category  "coding". Two categories don't have a hierachy.
Then I watch a video named "python learning" in YouTube website, its title will be like "python learning - youtube.com", matching two different categories. So where will it be?

**Comment by johan-bjareholt** [comment, marked as answer]
https://docs.activitywatch.net/en/latest/features/categorization.html#rules

> If there are multiple categories matching the same event then the deepest sub-category will be chosen.

If the depth is the same, the behaviour is undefined. But if you are lucky, it might choose the one lowest in the web-ui.

**Reply by NanGuoBean** [reply]
Thanks a lot! I haven't fully understood this rule. 
But now I know how to catagorize events that doesn't belong to some  specific categories!😁
👍👍👍
